definitions/tools/filter_sv_vcf.cwl

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+
+baseCommand: ["/bin/bash","filter_sv_vcf_read_support.sh"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 4000
+    - class: DockerRequirement
+      dockerPull: "mgibio/bcftools-cwl:1.9"
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: "filter_sv_vcf_read_support.sh"
+        entry: |
+          set -eou pipefail
+          
+          input_vcf="$1"
+          output_vcf="$2"
+          paired_count="$3"
+          paired_perc="$4"
+          split_count="$5"
+          split_perc="$6"
+          vcf_source="$7"
+
+          if [ "$vcf_source" == "smoove" ]; then
+            echo "Running filter for smoove vcf"
+            filter_expression="(AS >= $split_count && (AS / (RS+AS) >= $split_perc)) && (AP >= $paired_count && (AP / (AP+RP) >= $paired_perc))"
+          elif [ "$vcf_source" ==  "manta" ]; then
+            echo "Running filter for manta vcf"
+            filter_expression="(SR[0:*]=\".\" || (SR[0:1] >= 2 && (SR[0:1] / (SR[0:0]+SR[0:1]) >= $split_perc))) && (PR[0:1] >= $paired_count && (PR[0:1] / (PR[0:0]+PR[0:1]) >= $paired_perc))"
```

Comment:
It looks like `$split_count` isn't used in the manta version.  I'd guess it goes here?
```suggestion
            filter_expression="(SR[0:*]=\".\" || (SR[0:1] >= $split_count && (SR[0:1] / (SR[0:0]+SR[0:1]) >= $split_perc))) && (PR[0:1] >= $paired_count && (PR[0:1] / (PR[0:0]+PR[0:1]) >= $paired_perc))"
```